{"uuid": "42856e47-675e-4bd0-b8bd-f4d8479d8616", "children": ["f2c57a62-257a-44a5-9866-09e8dc947764"], "befores": [{"name": "standard_login", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: chrome not reachable\n  (Session info: chrome=89.0.4389.82)\n\n", "trace": "  File \"c:\\users\\olesya.laptop-q6599ufh\\pycharmprojects\\final_project\\venv\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\olesya.laptop-q6599ufh\\pycharmprojects\\final_project\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\olesya.laptop-q6599ufh\\pycharmprojects\\final_project\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\Olesya.LAPTOP-Q6599UFH\\PycharmProjects\\final_project\\conftest.py\", line 23, in standard_login\n    app.login.auth(login, passwd)\n  File \"C:\\Users\\Olesya.LAPTOP-Q6599UFH\\PycharmProjects\\final_project\\pages\\login.py\", line 29, in auth\n    self.input_value(self._username_input(), email)\n  File \"C:\\Users\\Olesya.LAPTOP-Q6599UFH\\PycharmProjects\\final_project\\pages\\login.py\", line 20, in _username_input\n    return self.app.driver.find_element(*LoginLocators.USERNAME_INPUT)\n  File \"c:\\users\\olesya.laptop-q6599ufh\\pycharmprojects\\final_project\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 976, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n  File \"c:\\users\\olesya.laptop-q6599ufh\\pycharmprojects\\final_project\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\olesya.laptop-q6599ufh\\pycharmprojects\\final_project\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1615299561880, "stop": 1615299584225}], "start": 1615299561880, "stop": 1615299584295}
